File: PRAK108-2210817120007-NursyahnaPuteri.py
kelilingtaman = 5
jaraktempuh = 14
phi = 3.14
kelilinglingkaran = jaraktempuh/kelilingtaman
jarijari = (kelilinglingkaran/(phi*2))

print("Diketahui :")
print("Pak Dengklek mengelilingi taman = {} putaran".format (kelilingtaman))
print("Jarak tempuh Pak Dengklek = {} kilomemeter".format (jaraktempuh))
print("Jawaban :")
print("Jari-jari taman yang dikelilingi Pak Dengklek adalah {} kilometer".format (round (jarijari,2)))

File: PRAK108-2210817120007-NursyahnaPuteri.c
#include <stdio.h>
int main () {

    float jaraktempuh, kelilingtaman, jarijari, phi, kelilinglingkaran;
    kelilingtaman = 5;
    jaraktempuh = 14;
    phi = 3.14;
    kelilinglingkaran = jaraktempuh/kelilingtaman;
    jarijari = (kelilinglingkaran/(phi*2));

    printf ("Diketahui : \n");
    printf ("Pak Dengklek mengelilingi taman = %.f putaran\n", kelilingtaman);
    printf ("Jarak tempuh Pak Dengklek = %.f kilomemeter\n",jaraktempuh);
    printf ("\n");
    printf ("Jawaban : \n");
    printf ("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f kilometer\n", jarijari);
}